src/components/DappBrowser/utils.ts

```diff
@@ -39,9 +43,11 @@ export const normalizeUrlWorklet = (url: string): string => {
   if (!url) {
     return '';
   }
-  if (url === RAINBOW_HOME) {
+
+  if (url === RAINBOW_HOME || url.startsWith('blob:') || url.startsWith('data:')) {
     return url;
   }
+
```

Comment:
@christianbaroni does it make more sense to return an empty string for `url.startsWith('blob:') || url.startsWith('data:')`?

Reply:
Do you know where the `blob:` / `data:` URLs are coming from?

Reply:
Asking because it may make sense to prevent these URLs from being loaded before they ever hit this function.